src/components/NewLocationPage/ChartFooter/MetricChartFooter.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from 'react';
+import {
+  Footer,
+  ButtonContainer,
+  FooterText,
+  AboutText,
+  ModalButton,
+} from './ChartFooter.style';
+import { useBreakpoint } from 'common/hooks';
+import NewDialog from 'components/NewDialog/NewDialog';
+import { Link } from 'cms-content/modals';
+
+export interface AboutModalProps {
+  header: string;
+  body: string;
+  links: Link[];
+}
+
+const MetricChartFooter: React.FC<{
+  footerText: string | null;
+  shareButton: any;
```

Comment:
want to render the share button directly in `MetricChartFooter` rather than passing it in as a prop? (and instead pass in the metric-specific share button props [that start here](https://github.com/covid-projections/covid-projections/blob/develop/src/components/LocationPage/ChartBlock.tsx#L69))